controllers/tasks.js

```diff
@@ -53,15 +53,23 @@ const fetchTasks = async (req, res) => {
  */
 const getUserTasks = async (req, res) => {
   try {
-    const { status } = req.query
+    let { status } = req.query
     const { username } = req.params
     let allTasks = []
 
-    if (status && !Object.values(TASK_STATUS).includes(status)) {
+    if (status && !Object.keys(TASK_STATUS).includes(status.toUpperCase())) {
       return res.boom.notFound('Status not found!')
     }
 
-    allTasks = await tasks.fetchUserTasks(username, status ? [status] : [])
+    if (status) {
+      if (status === 'active') {
+        status = ['active', 'blocked', 'pending', 'IN_PROGRESS', 'BLOCKED', 'SMOKE_TESTING']
```

Comment:
Can we have this array made into a variable and then calling this variable here, that will help us understand what this array is about